config/default.yml

```diff
@@ -212,6 +212,20 @@ Style/CollectionMethods:
     detect: 'find'
     find_all: 'select'
 
+# Use ` or %x around command literals.
+Style/CommandLiteral:
+  EnforcedStyle: mixed
```

Comment:
According to the style guide, this should probably be changed to `backticks`, right?